src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -353,123 +353,12 @@
-   (let [more-params
-         (httpfn/extract-params (get-in db [::db/browser :more-link]))
-         address (get-in db [::db/browser :address])]
-     ;; Push current address into stack
-     {:db (update-in db [::db/browser :back-stack] conj address)
-      :dispatch [:browser/load-xapi {:params more-params}]})))
-
-(re-frame/reg-event-fx
- :browser/back
- global-interceptors
- (fn [{:keys [db]} _]
-   ;; Pop most recent from stack
-   (let [back-stack (get-in db [::db/browser :back-stack])
-         back-params (httpfn/extract-params (peek back-stack))]
-     {:db (update-in db [::db/browser :back-stack] pop)
-      :dispatch [:browser/load-xapi {:params back-params}]})))
-
-(re-frame/reg-event-fx
- :browser/add-filter
- global-interceptors
- (fn [{:keys [db]} [_ param-key param-value]]
-   (let [address (get-in db [::db/browser :address])
-         params (-> (httpfn/extract-params address)
-                    (dissoc "from" "limit")
-                    (assoc param-key param-value))]
-     ;; Clear back-stack
-     {:db (assoc-in db [::db/browser :back-stack] [])
-      :dispatch [:browser/load-xapi {:params params}]})))
-
-(re-frame/reg-event-fx
- :browser/clear-filters
- global-interceptors
- (fn [{:keys [db]} _]
-   {;; Clear back-stack and reset query
-    :db (assoc-in db [::db/browser :back-stack] [])
-    :dispatch [:browser/load-xapi]}))
-
-(re-frame/reg-event-fx
- :browser/update-credential
- global-interceptors
- (fn [{:keys [db]} [_ key]]
-   (let [credential (first (filter #(= key (:api-key %))
-                                   (::db/credentials db)))]
-     (when credential
-       ;; Clear backstack and limit and filters
-       {:db (update-in db [::db/browser] assoc
-                       :credential credential
-                       :back-stack []
-                       :batch-size 10)
-        :dispatch [:browser/load-xapi]}))))
-
-(re-frame/reg-event-fx
- :browser/refresh
- global-interceptors
- (fn [{:keys [db]} _]
-   (when (get-in db [::db/browser :credential])
-     ;; Clear backstack
-     {:db (assoc-in db [::db/browser :back-stack] [])
-      :dispatch [:browser/load-xapi]})))
-
-(re-frame/reg-event-fx
- :browser/update-batch-size
- global-interceptors
- (fn [{:keys [db]} [_ batch-size]]
-   ;; Clear from and limit
-   (let [address (get-in db [::db/browser :address])
-         params (-> (httpfn/extract-params address)
-                    (dissoc "from" "limit"))]
-     ;; update batch to new size and clear back-stack
-     {:db (update-in db [::db/browser] assoc
-                     :batch-size batch-size
-                     :back-stack [])
-      :dispatch [:browser/load-xapi {:params params}]})))
-
-
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Api Key Management
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+;; TODO: This is a separate page; consider a separate handler namespace?
```

Comment:
This is a question I have: should credential, admin account, and actor delete handlers and subs also have their own namespaces?

Reply:
Yeah these other handler sets don't really need separate namespaces since they don't have oodles of handlers _and_ subs, unlike the ones that got moved.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/90/commits/7c78e067bcb8395a7594e924990d3e3a40e47479